### DISCUSSION TITLE
Autoscaling nodes with labels

### DISCUSSION THREAD

**Discussion by AnttiKarvinen** [vitobotta/hetzner-k3s#267] [Q&A] [answered]
Is there any way to make use of labels in these autoscaling nodes.
- name: nbg1-cpx41
  instance_type: cpx41
  instance_count: 0
  location: nbg1
  labels:
    - key: spark-role
      value: executor
   autoscaling:
    enabled: true
    min_instances: 0
    max_instances: 5
or example this piece of configuration just autostarts nodes but doesn't make labels to them.

**Comment by vitobotta** [comment, marked as answer]
Unfortunately labels are not yet supported by the cluster autoscaler for Hetzner, see https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/hetzner/README.md.

**Comment by AnttiKarvinen** [comment]
Thank you, we'll manage another way.
